Algorithms/GEMETD.py

```diff
@@ -0,0 +1,56 @@
+from Algorithms.ETDLB import ETDLB
+from Algorithms.BaseTD import BaseTD
+import numpy as np
+
+
+class GEMETD(ETDLB):
+    """
+    An ETD(0) implementation that uses GEM (aka GTD2(0) with x and x_p switched) to estimate emphasis.
+    """
+    def __init__(self, task, **kwargs):
+        super().__init__(task, **kwargs)
+        self.beta = self.task.GAMMA
+        self.gem_alpha = self.get('gem_alpha')  # Step size for GEM weights.
```

Comment:
self.alpha is already defined in BaseTD and its child ETDLB. I think we do not need to look for alpha again. We just need to use self.alpha. Does it make sense?

Reply:
GEM introduces two weight vectors beyond the existing weight vector `self.w`. It uses `self.gem_alpha` as the step size for the two new weight vectors, leaving `self.alpha` as the step size for `self.w`, so I think `gem_alpha` is still needed.

Reply:
Got it. I will fix this.